Zend/zend_interfaces.stub.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+interface Traversable {}
+
+interface IteratorAggregate extends Traversable
+{
+    /** @return Traversable */
+    function getIterator();
+}
+
+interface Iterator extends Traversable
+{
+    function current();
+
+    /** @return void */
+    function next();
+
+    function key();
+
+    /** @return bool */
+    function valid();
+
+    /** @return void */
+    function rewind();
+}
+
+interface ArrayAccess
+{
+    function offsetExists($offset);
+
+    /* actually this should be return by ref but atm cannot be */
+    function offsetGet($offset);
+
+    function offsetSet($offset, $value);
+
+    function offsetUnset($offset);
+}
+
+interface Serializable
+{
+    /** @return string */
+    function serialize();
+
+    function unserialize(string $serialized);
```

Comment:
Because it's part of an interface rather than a class, this type may cause BC issues. As Serializable needs to be phased out anyway, I would drop it.

Reply:
Nevermind, I'm not sure why I thought that `interface` is relevant here.